sycl/test/esimd/simd_view.cpp

```diff
@@ -69,6 +69,19 @@ SYCL_ESIMD_FUNCTION void test_simd_view_copy_ctor() {
   auto v0_view_copy(v0_view);
 }
 
+// test construction from vector.
+SYCL_ESIMD_FUNCTION void test_simd_view_from_vector() {
+  simd<int, 16> s = 0;
```

Comment:
please a test case for length 1

Reply:
Added here: 33eede79f2ca2d92d21dca4b85ca2227bebeb47d